packages/shared/forks/ReactFeatureFlags.www-modern.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import typeof * as FeatureFlagsType from 'shared/ReactFeatureFlags';
+import typeof * as FeatureFlagsShimType from './ReactFeatureFlags.www-modern';
+
+// Re-export dynamic flags from the www version.
+export const {
+  debugRenderPhaseSideEffectsForStrictMode,
+  disableInputAttributeSyncing,
+  enableTrustedTypesIntegration,
+  deferPassiveEffectCleanupDuringUnmount,
+  warnAboutShorthandPropertyCollision,
+} = require('ReactFeatureFlags');
+
+export let enableUserTimingAPI = false;
+
+export const enableProfilerTimer = __PROFILE__;
+export const enableSchedulerTracing = __PROFILE__;
+export const enableSchedulerDebugging = true;
+
+export const replayFailedUnitOfWorkWithInvokeGuardedCallback = false;
+export const warnAboutDeprecatedLifecycles = true;
+export const disableLegacyContext = true;
+export const warnAboutStringRefs = false;
+export const warnAboutDefaultPropsOnFunctionComponents = false;
+export const disableSchedulerTimeoutBasedOnReactExpirationTime = false;
+
+export const enableTrainModelFix = true;
+
+export const exposeConcurrentModeAPIs = __EXPERIMENTAL__;
+
+export const enableSuspenseServerRenderer = true;
+export const enableSelectiveHydration = true;
+
+export const enableChunksAPI = __EXPERIMENTAL__;
+
+export const disableJavaScriptURLs = true;
+
+export function addUserTimingListener() {
+  throw new Error('Not implemented.');
+}
+
+export const enableDeprecatedFlareAPI = true;
```

Comment:
Can we add a comment to group the two sets up more? I’m worried people will add to the bottom of this list and it will get muddled up otherwise.

Reply:
This is 100% copy paste from the other file. I agree we should clean it up but let's do this separately from this PR.

Reply:
Sounds good.